src/main/resources/component-biz.conf

```diff
@@ -1246,6 +1246,11 @@ storage {
                 forceSSL = false
             }
         }
+
+        # Contains a list of known file extensions of server-sided scripting languages. This can be used
+        # to determine if the file-extension of a file-url contains the effective filename or if the path
+        # is just a script and we need to inspect the ContentDisposition (etc.) to obtain the actual file name
+        serverSidedScriptingExtensions = [ "php", "php3", "cgi", "asp", "aspx", "jsp", "jspx", "cfm", "cfml" ]
```

Comment:
Perhaps an edge case, but I've seen XML used to render pages.
An XSLT file can be declared and the browser will run the xsl transformation locally, rendering the results.

Reply:
An xml could also be a valid source file :-/